### PR TITLE
BE-54 Add Elasticsearch integration

### PR DESCRIPTION
**Pull request by OliverKanat**

### CODE REVIEW

**Review by Scr0ll — DISMISSED** [review]

File: antiplag/antiplag/documents.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from django_elasticsearch_dsl.registries import registry
+from .models import Document as DocumentModel
+
+
+@registry.register_document
+class DocumentModelDocument(Document):
+    class Index:
+        # TODO: implement
+
+        # Name of the Elasticsearch index
```
Comment (MCFreddie777) [inline antiplag/antiplag/documents.py:11]: Are you planning on implementing this as a part of this task? Or we can merge it and implement later?

File: antiplag/antiplag/documents.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        name = 'documents'
+        # See Elasticsearch Indices API reference for available settings
+        #settings = {'number_of_shards': 1,
+        #            'number_of_replicas': 0}
+
+    class Django:
+        model = DocumentModel # The model associated with this Document
+
+        # The fields of the model you want to be indexed in Elasticsearch
+        fields = []
```
Comment (MCFreddie777) [inline antiplag/antiplag/documents.py:21]: Missing empty line.

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]
Reviewed. Can't see the changes tho, unless it's properly impelmented

File: antiplag/antiplag/documents.py (hunk 1 above)
Comment (MCFreddie777) [reply antiplag/antiplag/documents.py:11]: We agreed in Slack on implementing it in other task.

**Review by MCFreddie777 — APPROVED** [review]
Approved ✅

**Pull request merged**